consul/settings.sls

```diff
@@ -47,10 +47,14 @@
 {%- set servers = salt['mine.get'](server_target, 'network.get_hostname', targeting_method).values() %}
 
 # Create a list of servers that can be used to join the cluster
-{%- set join_server = None %}
-{%- for server in servers if server != nodename %}
-    {%- set join_server = server %}
-{%- endfor %}
+# Jinja vars are immutable in a loop so you have to do the 'do' trick to append to the list
+# http://stackoverflow.com/questions/17925674/jinja2-local-global-variable
+{% set servers = ['im', 'devopsdev', 'devopsdump' ] %}
```

Comment:
I'm assuming these servers are left over test code? If so, can you clean them out?